src/Alliteration.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace Nubs\RandomNameGenerator;
+
+/**
+ * Defines the standard interface for all the random name generators.
+ */
+class Alliteration implements Generator
+{
+    private $adjectives;
+    private $nouns;
+
+    /**
+     * Create a new instance of Alliteration
+     */
+    public function __construct()
+    {
+        $this->adjectives = file(__DIR__ . '/adjectives.txt');
+        $this->nouns = file(__DIR__ . '/nouns.txt');
+    }
+
+    /**
+     * Gets a randomly generated alliterative name.
+     *
+     * @return string A random name.
+     */
+    public function getName()
+    {
+        $letter = chr(rand(97, 122));
```

Comment:
Wouldn't perform as well, but this could be the following which may read nicer.:
```php
$letters = range('a', 'z');
$letter = $letters[array_rand($letters)];
```

Or perhaps,
```php
$letter = chr(rand(ord('a'), ord('z')));
```

Just trying to think of ways to not have `97` and `122` in there.